content/en/flagger/tutorials/keda-scaledobject.md

```diff
@@ -10,7 +10,7 @@ you can use any deployment strategy combined with any supported provider.
 
 ## Prerequisites
 
-Flagger requires a Kubernetes cluster **v1.16** or newer. For this tutorial, we'll need KEDA **2.71** or newer.
+Flagger requires a Kubernetes cluster **v1.16** or newer. For this tutorial, we'll need KEDA **2.7.1** or newer.
```

Comment:
i'm not sure whether Flagger will work with Kubernetes v1.16. if true, we need to go through all mentions of this and update it to the correct version. cc: @stefanprodan

Reply:
I applied the v1.21.0..v1.22.2 diff from https://github.com/fluxcd/flagger/tree/main/docs/gitbook

Reply:
yes, i'm aware, we would need to update the gitbook in the Flagger repo as well

Reply:
let's not treat this as a blocker for this PR, since this is just syncing changes with upstream

Reply:
We need to update Flagger’s docs and set the Kubernetes version to the oldest version that’s not EOL.